pycytominer/cyto_utils/cells.py

```diff
@@ -846,3 +775,67 @@ def aggregate_profiles(
             )
         else:
             return aggregated
+
+
+def _sqlite_strata_conditions(df, dtypes, n=1):
+    """Given a dataframe where columns are merge_cols and rows are unique
+    value combinations that appear as aggregation strata, return a list
+    of strings which constitute valid SQLite conditional statements.
+
+    Parameters
+    ----------
+    df : pandas.core.frame.DataFrame
+        A dataframe where columns are merge_cols and rows represent
+        unique aggregation strata of the compartment table
+    dtypes : Dict[str, str]
+        Dictionary to look up SQLite datatype based on column name
+    n : int
+        Number of rows of the input df to combine in each output
+        conditional statement. n=1 means each row of the input will
+        correspond to one string in the output list. n=2 means one
+        string in the output list is comprised of two rows from the
+        input df.
+
+    Returns
+    -------
+    grouped_conditions : List[str]
+        A list of strings, each string being a valid SQLite conditional
```

Comment:
can you add a couple words describing how this is meant to be used?

"being a valid SQLite conditional to facilitate chunked aggregation?" 🤷

Reply:
Haha yeah, I can add a bit.  Really what I mean is that it will return a list of strings (called `conditions`), where each of the strings can be used in an SQL statement like this:
```python
f"select {cols} from {compartment} where {condition}"
```
Here `condition` is one element of the list of strings `conditions`, and the above SQL statement will grab a single chunk for aggregation.  So by iterating over the conditions in `grouped_conditions` and making SQL queries, we get the whole table back in chunks:
```python
for condition in conditions:
    do_sql_query(f"select {cols} from {compartment} where {condition}")
```